stressberry/cli.py

```diff
@@ -164,4 +215,35 @@ def _get_parser_plot():
         default=None,
         help="limits for the temperature (default: data limits)",
     )
+    parser.add_argument(
+        "-d",
+        "--dpi",
+        type=int,
+        default=None,
+        help="image resolution in dots per inch when written to file",
+    )
+    parser.add_argument(
+        "-f",
+        "--frequency",
+        help="plot CPU core frequency (single input files only)",
+        action="store_true",
+    )
+    parser.add_argument(
+        "-l",
+        "--freq-lims",
+        type=float,
+        nargs=2,
+        default=None,
+        help="limits for the frequency scale (default: data limits)",
+    )
+    parser.add_argument("--hide-legend", help="do not draw legend", action="store_true")
+    parser.add_argument(
+        "--transparent",
```

Comment:
Rename to `"--not-transparent` and add `dest="transparent"`  to make it come out right.